### PR TITLE
[FrameworkBundle][RateLimiter] Update typos in config

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| Branch?       | 5.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

This is just a super minor

### CODE REVIEW

**Review by bykclk — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -1843,7 +1843,7 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
                                         ->defaultNull()
                                     ->end()
                                     ->enumNode('policy')
-                                        ->info('The rate limiting algorithm to use for this rate')
+                                        ->info('The algorithm to used by this limiter')
```
Comment (derrabus) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]:
```suggestion
                                        ->info('The algorithm to be used by this limiter')
```

**Comment by Nyholm** [comment]
Thank you for the reviews

**Review by derrabus — APPROVED** [review]

**Comment by wouterj** [comment]
Thanks Tobias!

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**